projects/models.py

```diff
@@ -0,0 +1,281 @@
+from django import forms
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+MAX_TITLE_LENGTH = 255
+
+
+class ProjectLeader(models.Model):
+    """
```

Comment:
Is the PorjectLeader or MediaEducationSupervisor an OK employee? And if so, should it be linked to the corresponding user?